src/shared/ui/survey/AgeSelector/AgeSelector.tsx

```diff
@@ -0,0 +1,44 @@
+import { FC, useMemo } from 'react';
+
+import { useTranslation } from 'react-i18next';
+
+import AgeSelectorConfigType from './types';
+import Dropdown from '../../Dropdown';
+
+type DropdownOnChangeValue = {
+  value: string | number;
+};
+
+type Props = {
+  config: AgeSelectorConfigType;
+  value?: string | number;
+  onChange: (value: DropdownOnChangeValue) => void;
```

Comment:
DropdownOnChangeValue - maybe DropdownChangeValue or even DropdownValue, or SelectedAgeValue
Please think for better name

Reply:
Fixed